src/core/lib/gpr/arena.cc

```diff
@@ -74,11 +85,6 @@ void gpr_arena_init() {
   gpr_once_init(&g_init_strategy_once, set_strategy_from_env);
 }
 
-// Uncomment this to use a simple arena that simply allocates the
-// requested amount of memory for each call to gpr_arena_alloc().  This
-// effectively eliminates the efficiency gain of using an arena, but it
-// may be useful for debugging purposes.
-//#define SIMPLE_ARENA_FOR_DEBUGGING
```

Comment:
Isn't the comment helpful?

Reply:
The comment still exists; it is just moved to the header now.